api/tests/__init__.py

```diff
@@ -1,3 +1,8 @@
 import httpretty
 
+import koiki
+
 httpretty.enable(allow_net_connect=False)
+
+koiki.host = 'https://testing_host'
```

Comment:
Good idea to decouple this since test would end up repeating it a lot.

Reply:
It took quite a while to get the idea working but I think this is far better, yes.